### ISSUE TITLE
Blame action doesn't work in position:view

### ISSUE THREAD

**Issue by phil294** [phil294/git-log--graph#77]
https://github.com/phil294/git-log--graph/issues/40#issuecomment-1962294235

> Sorry for asking here. I'm not sure if this is an issue or if I'm not understanding it right.
I noticed that the editor's context menu has a Blame: Focus last commit at cursor position in graph view option (I was wondering where it came from until I realized here that it comes from git-log-graph).
It doesn't seem to do anything when I click on it. Sounds like it should activate the graph view and highlight that specific commit but it doesn't do that.
I tried the command git-log--graph.blame-line, nothing reacted to it.
I have set "git-log--graph.position": "view" and the view is placed in the bottom panel of the editor. The line at cursor position does have a blame.
Other commands like git-log--graph.start can work but only this one doesn't work. Does it require any conditions or settings?

**Comment by Amanoki** [comment]
> @Amanoki no it doesn't require anything, and since v0.1.14 (yesterday), it should also always work reliably. This looks like a bug, I'll just make a new issue out of this #77

Yes looks like it is supposed to be working but it has never worked for me since I installed the extension a couple weeks ago until now.
It also doesn't work under `"git-log--graph.position": "editor"`
My VS Code is the latest version. No other useful info to offer beyond these.

**Comment by stunited-thien** [comment]
same here, I didn't see any git blame feature

**Comment by hansu** [comment]
Doesn't work for me either.
And I would suggest to change the name of the context menu entry that one know that this belongs to GitLG.
Maybe 
`Focus last commit at cursor position in GitLG`
or
`GitLG: Focus last commit at cursor position in graph view `

**Comment by phil294** [comment]
@hansu will do.

I'll have a look at this (and the various other issues in just a few days).

**Comment by hansu** [comment]
No rush, just stumbled over this context menu entry and wondered where it originates from (the fact that it didn't work didn't make it easier to figure that out 😄 )
